frontend/src/pages/connectionTypes/manage/ManageConnectionTypePage.tsx

```diff
@@ -131,11 +131,11 @@ const ManageConnectionTypePage: React.FC<Props> = ({ prefill, isEdit, onSave })
       >
         {isEdit ? (
           <PageSection variant="light" className="pf-v5-u-pt-0">
-            <Alert
-              isInline
-              variant="warning"
-              title="Editing this connection will not affect existing connections of this type."
-            />
+            <Alert isInline variant="warning" title="Consider renaming before saving changes">
+              Editing this connection type does not affect existing connections of this type. To
+              keep track of previous versions of this connection type, consider renaming it before
+              saving changes.{' '}
+            </Alert>
```

Comment:
There's a trailing `{' '}` that is unnecessary.

The copy here doesn't make sense because renaming in this form does not create a new copy. The user must duplicate the connection type prior to editing.

Reply:
I think the thought is that if you rename it the existing connections of this type will still be named the old name while new names with these edits will have a different name. At least then the users could tell them apart even though the old connection type name would no longer exist.